### PR TITLE
[#1113] Ignore Records With Combined In Title

### LINKED ISSUE #1113: Ignore Records With "- Combined" In Title from Export
Don't export records that end with "- Combined".

### PR DESCRIPTION
**Pull request by Meowcenary**
Ignore records with either "- combined" or "- Combined" in the title. closes #1113

### CODE REVIEW

**Comment by Meowcenary** [comment]
Once this is reviewed I'll start a new release

**Review by mattbaumann1 — APPROVED** [review]

File: spec/models/invenio_rdm_record_converter_spec.rb [hunk 1]
```diff
@@ -226,7 +226,7 @@
               "scheme": "doi",
               "relation_type": {"id": "isRelatedTo"}
             }],
-          "sizes": ["#{generic_file.page_count} pages"],
+            "sizes": ["#{generic_file.page_count.shift} pages"],
```
Comment (fenekku) [inline spec/models/invenio_rdm_record_converter_spec.rb:229]: minor: extra indent

File: spec/models/invenio_rdm_record_converter_spec.rb [hunk 2]
```diff
@@ -289,6 +289,9 @@
   let(:invenio_rdm_record_converter) {
     described_class.new(generic_file, collection_store.data, role_store.data)
   }
+  let(:unexportable_generic_file_upper) {  make_generic_file_with_content(user, title: ["Test file - Combined"]) }
```
Comment (fenekku) [inline spec/models/invenio_rdm_record_converter_spec.rb:292]: ultra minor: extra space

**Review by fenekku — APPROVED** [review]
Thanks. The comments are super minor.

**Comment by Meowcenary** [comment]
> Thanks. The comments are super minor.

Thanks for catching those. Merging this in now and beginning release work!

**Pull request merged**